docs/source/_templates/index.rst

```diff
@@ -1,66 +1,72 @@
     :title: The landing page of the dashboard of the application.
 
 To find the application you're looking for, you can navigate through the different app pages (3), use tags (4), or utilize the search bar (5).
 
-Once the desired application is found, simply click on it to initiate the process. This will start the smallest instance to run the SEPAL sandbox (:code:`t1`). Please refer to the next section to start a specific instance manually.
+Once the desired application is found, select it to initiate the process. This will start the smallest instance to run the SEPAL sandbox (:code:`t1`). 
+
+Refer to the next section to start a specific instance manually.
 
 .. note:: 
 
-    Applications that include a :icon:`fas fab-google` in their descriptor require a connection to Google Earth Engine (GEE) to run. Please refer to `Connect GEE and SEPAL <../setup/gee.html#connection-between-gee-and-sepal>`__ for more information.
+    Applications that include a :icon:`fas fab-google` in their descriptor require a connection to Google Earth Engine (GEE) to run. For more information, see `Connect GEE and SEPAL <../setup/gee.html#connection-between-gee-and-sepal>`__.
     
 .. note::
 
-    An application relies on numerous tools and systems to run. Since it can take up to five minutes to start, stay patient and avoid rushing. If the application doesn't start after five minutes, it could simply be an issue with the Amazon cloud connection. We recommend closing the tab and starting again. If you do experience a major bug or crash, please report the issue in the specific app issue tracker or directly in SEPAL.
+    An application relies on numerous tools and systems to run. Since it can take up to five minutes to start, stay patient and avoid rushing. If the application doesn't start after five minutes, it could simply be an issue with the Amazon cloud connection; we recommend closing the tab and starting again. In the event of a major bug or crash, please report the issue in the specific app issue tracker or directly in SEPAL.
     
 .. important:: 
 
     To avoid disturbances in the application workflow, avoid stopping instances manually and disconnecting from SEPAL. While completed work will be available in SEPAL folders, you will need to restart the process in order to finish.
 
-
 Start instance manually
 -----------------------
 
-Some applications require more powerful instances than the default :code:`t1`. If the app documentation requires the user to start a specific instance type (e.g. a GPU machine :code:`g4` or :code:`g8`), follow these steps prior to initiating the application. 
+Some applications require more powerful instances than the default :code:`t1`. If the app documentation requires the user to start a specific instance type (e.g. a GPU machine :code:`g4` or :code:`g8`), follow these steps prior to initiating the application:
 
-#.  Go to the SEPAL terminal and wait for the instance selector to start.
```

Comment:
any reason why you replaced the "#" (which is automatically numbered by Sphinx by hard coded numbers ?